ang/test/civicase/Dashboard.spec.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-env jasmine */
+
+(function (_) {
+  describe('civicaseDashboardController', function () {
+    var $controller, $rootScope, $scope;
+
+    beforeEach(module('civicase', 'civicase.data'));
+
+    beforeEach(inject(function (_$controller_, _$rootScope_) {
+      $controller = _$controller_;
+      $rootScope = _$rootScope_;
+
+      $scope = $rootScope.$new();
+      $scope.$bindToRoute = jasmine.createSpy('$bindToRoute');
+
+      spyOn(CRM, 'checkPerm');
+    }));
+
+    describe('filter', function () {
+      describe('when user has permission to view all cases', function () {
+        beforeEach(function () {
+          CRM.checkPerm.and.returnValue(true);
+          initController();
+        });
+
+        it('shows the All Cases filter option', function () {
```

Comment:
If we are using \` for highlighting [line:50](https://github.com/compucorp/uk.co.compucorp.civicase/pull/73/files#diff-dcae1024db1019fa8c8b6afd553bbf46R51). Should we use here too?
```
      it('shows the `All Cases` filter option', function () {
```